src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -46,6 +46,18 @@
 register_cli_argument('resource feature list', 'resource_provider_namespace', CliArgumentType(options_list=('--namespace',), required=False, help=_PROVIDER_HELP_TEXT))
 register_cli_argument('resource feature', 'feature_name', CliArgumentType(options_list=('--name', '-n'), help='the feature name'))
 
+register_cli_argument('resource policy', 'resource_group_name', arg_type=resource_group_name_type, completer=get_resource_group_completion_list)
+register_cli_argument('resource policy', 'policy_definition_name', options_list=('--name', '-n'), help='name of policy definition')
+register_cli_argument('resource policy create', 'rules', help='JSON formatted string or a path to a file with such content')
```

Comment:
We can discuss it, but `json` format is much better under the context. 
The conditional policy logic is rather awkward to split into individual argument. Even we do, it is much harder to use. The schema is [here](https://azure.microsoft.com/en-us/documentation/articles/resource-manager-policy/) for reference. After read it, I understood using JSON content is the only feasible option. That is why ARM went with it for node cli.